src/Terminus.php

```diff
@@ -2,47 +2,88 @@
 
 namespace Pantheon\Terminus;
 
-use Symfony\Component\Console\Application;
-use Symfony\Component\Console\Input\InputOption;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use Robo\Runner;
 
-class Terminus extends Application
+class Terminus
 {
+    /**
+     * @var string[]
+     */
+    private $args;
+    /**
+     * @var string
+     */
+    private $commands_directory;
+    /**
+     * @var \Robo\Runner
+     */
+    private $runner;
+    /**
+     * @var string
+     */
+    private $top_namespace = 'Pantheon\Terminus\Commands';
 
     /**
-     * @inheritdoc
+     * Runner constructor
+     *
+     * @param array $arguments Arguments from the command line
      */
-    public function __construct()
+    public function __construct(array $arguments)
     {
-        $config = new Config();
-        parent::__construct('Terminus', $config->get('version'));
+        $this->args = $arguments;
+        $this->commands_directory = __DIR__ . '/Commands';
+        $this->setRunner();
     }
 
     /**
-     * @inheritdoc
+     * Runs the instantiated Terminus application
+     *
+     * @return integer The exiting status code of the application
      */
-    protected function getDefaultHelperSet()
+    public function run()
     {
-        $helper_set = parent::getDefaultHelperSet();
-        //$helper_set->set(new Pantheon\Terminus\Helpers\FileHelper());
-        return $helper_set;
+        $config = new Config();
+        return $this->runner->execute($this->args, null, 'Terminus', $config->get('version'));
     }
 
     /**
-     * @inheritdoc
+     * Adds command files to the application
+     *
+     * @return void
      */
-    protected function getDefaultInputDefinition()
+    private function setRunner()
     {
-        $definition = parent::getDefaultInputDefinition();
-        $definition->addOptions(
-            [
-            new InputOption(
-                '--yes',
-                '-y',
-                InputOption::VALUE_NONE,
-                'Answer yes to all prompts'
-            ),
-            ]
+        $discovery = new CommandFileDiscovery();
+        $discovery->setSearchPattern('*Command.php')->setSearchLocations(['.']);
+        $command_classes = $this->fixNames($discovery->discover($this->commands_directory, $this->top_namespace));
+        $this->runner = new Runner($command_classes);
```

Comment:
@greg-1-anderson I changed things around to make use of Robo's Runner class, but I'm still getting NullOutput objects. :(